screwdriver.yaml

```diff
@@ -0,0 +1,15 @@
+shared:
+    image: golang
+    environment:
+        GOPATH: /sd/workspace
+
+jobs:
+    main:
+        requires: [~pr, ~commit]
+        steps:
+            - install: go mod download
+            - vet: go vet ./...
+            - gofmt: ret=$(find . -name '*.go' | xargs gofmt -d) && echo "$ret" && test -z "$ret"
+            - gover: go version
```

Comment:
I think showing version should be placed at first step or before using `go`.

Reply:
make sense. i fixed